ros/src/computing/perception/detection/lidar_tracker/packages/lidar_imm_ukf_pda_track/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 2.8.3)
-project(lidar_imm_ukf_pda_track)
+project(imm_ukf_pda_track)
```

Comment:
@aohsato @cirpue49 Finally, the name should be imm_ukf_pda because only the packages in Detection have the naming convention {lidar,vision}_name_{detector,tracker}. For example, ndt_matching is not lidar_ndt_matching_localizer. However changing name need not be in this pull request because this pull request is not for the purpose.